ddocs/medic-client/views/reports_by_form_year_week_places/map.js

```diff
@@ -1,73 +1,24 @@
 // data record must adhere to property name of `week` or `week_number`
 function(doc) {
```

Comment:
Thanks. It took a lot of pulling apart to work out what we didn't need. Also some stuff was duplicated between the key and the value. Hopefully I haven't missed anything and regressed...